### DISCUSSION TITLE
How can I export the filtered dataframe in the Data tab ?

### DISCUSSION THREAD

**Discussion by zhentai-SMU** [Kanaries/pygwalker#596] [Q&A]
Hello, I am a beginner for the pygwalker. I would like to ask if exporting the filtered dataframe on the interface is possible. Intuitively, I think there should be an export button at this location
![image](https://github.com/user-attachments/assets/2ef6d9dc-f3c3-45de-a863-38354c60b259)

**Comment by longxiaofei** [comment]
@zhentai-SMU Thanks for your feedback.

This looks like a very useful feature and we are planning to implement it next week.

**Reply by zhentai-SMU** [reply]
That's awesome! Thanks for your GREAT work!

**Reply by riteshhub** [reply]
@longxiaofei is it available now?
